parley_bench/README.md

```diff
@@ -0,0 +1,19 @@
+# Parley Benchmarks
+
+A suite of benchmarks used to evaluate Parley performance.
```

Comment:
We should link to Tango somewhere here, so people have a chance to understand what's actually going on.

It looks like the right link is <https://github.com/bazhenov/tango>. Something like:
```md
This uses [Tango](https://github.com/bazhenov/tango) to perform paired benchmarking.
```

Reply:
I also just saw that Tango would require manual changes and nightly on Windows - we should also document that (even if it's just "Windows might require additional setup, see Tango's docs for details")